### DISCUSSION TITLE
Documenting a video popup with a video set to a certain (start) time

### DISCUSSION THREAD

**Discussion by djmonzyk** [biati-digital/glightbox#410] [Announcements]
I know the plyr plugin is supposed to be used if I want to go to a certain timestamp in a YouTube video from a link. However, it isn't documented very well. Is there any way an example could be provided? I'm trying to create links that will fire the different chapters of a single YouTube video in a glightbox.

I think others would benefit from a documented example.

Thanks!

**Comment by gingerchew** [comment]
That sounds like you would need to use some of the events GLightbox offers. I think `slide_before_load` is what you are looking for. I'm working on trying to collect some examples that are frequently asked for, so if you are able to pull one together in a CodePen, I'd love to add it :)
